# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/mp4/MP4Interval.h
# Commit: 89c241451a32
# Full Hash: 89c241451a32d362967315a0e2424fcce824208b
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-05-18 03:51:42
# Regressor Bug: 1817997
# File Overlap Count: 3
# Description:
#   Bug 1817997 - Update the MP4 demuxer to use TimeUnit based on the internal MP4 time base. r=alwu
#   
#   This still gets the initial time value from mp4parse-rust, that is in
#   microseconds. mp4parse-rust has been updated to expose real time, and will be
#   updated later.
# ==============================================================================

diff -r 66e4a26e1ebe -r 89c241451a32 dom/media/mp4/MP4Interval.h
--- a/dom/media/mp4/MP4Interval.h	Wed May 17 15:47:04 2023 +0000
+++ b/dom/media/mp4/MP4Interval.h	Wed May 17 15:47:04 2023 +0000
@@ -12,7 +12,7 @@
 
 template <typename T>
 struct MP4Interval {
-  MP4Interval() : start(0), end(0) {}
+  MP4Interval() : start{}, end{} {}
   MP4Interval(T aStart, T aEnd) : start(aStart), end(aEnd) {
     MOZ_ASSERT(aStart <= aEnd);
   }